test/res.redirect.js

```diff
@@ -19,6 +19,42 @@ describe('res', function(){
       .expect(302, done)
     })
 
+    it('should throw an error if the url is missing', function(done){
+      var app = express();
+
+      app.use(function (req, res) {
+        res.redirect(undefined)
+      })
+
+      request(app)
+      .get('/')
+      .expect(500, /url argument is required to res.redirect/, done)
+    })
+
+    it('should throw an error if the url is not a string', function(done){
+      var app = express();
+
+      app.use(function (req, res) {
+        res.redirect(['http://google.com'])
+      })
+
+      request(app)
+      .get('/')
+      .expect(500, /res.redirect: url must be a string/, done)
+    })
+
+    it('should throw an error if the status is not a number', function(done){
+      var app = express();
+
+      app.use(function (req, res) {
+        res.redirect("300", 'http://google.com')
```

Comment:
```suggestion
        res.redirect("300", 'https://google.com')
```
http -> https

Reply:
Not bad, but also just to be clear it is not necessary in the tests.